### DISCUSSION TITLE
Highlight action items in ERESOLVE error msg?

### DISCUSSION THREAD

**Discussion by ruyadorno** [npm/rfcs#334] [RRFC]
We still get lots of reports from folks using the cli and getting frustrated when hitting an `ERESOLVE` error, missing the possible action items described in the error message.

> Example of an ERESOLVE error message:
> ```sh
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! Found: abbrev@1.0.9
> npm ERR! node_modules/abbrev
> npm ERR!   peer abbrev@"~1.0.0" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer abbrev@"=1.1.1" from a@undefined
> npm ERR! a
> npm ERR!
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> npm ERR!
> npm ERR! See /<path>/.npm/eresolve-report.txt for a full report.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /<path>/.npm/_logs/2021-03-03T03_10_06_004Z-debug.log
> ```

While the message is very accurate, maybe it can use some different UI elements to help highlight the possible solutions:
- `npm install --force`
- `npm install --legacy-peer-deps`

This is a difficult balance act to get right since we don't want **all** users to ignore the error message and run in these modes but rather unblock users that are not in capacity of fixing their install tree.

**Comment by wesleytodd** [comment]
I think the example from the twitter thread leading to this, I think that the issue here was that it was miss-identified despite it being in the output. When I ran it, I noticed that the dep lines which conflicted were bold, so I wonder if just instead making the action items bold would solve the confusion?

**Comment by ruyadorno** [comment]
From the 17th March OpenRFC call:
- let's make a block with more explicit looking npm commands along with the flags so that users can copy and paste
- one possible improvement is to make these suggestions an interactive prompt so that users can be unblocked when hitting it (maybe same for the "did you mean" feature)
